app/controllers/compute_attributes_controller.rb

```diff
@@ -1,12 +1,14 @@
 class ComputeAttributesController < ApplicationController
   include Foreman::Controller::Parameters::ComputeAttribute
+  include Foreman::Controller::NormalizeScsiAttributes
 
   def new
     @set = ComputeAttribute.new(:compute_profile_id => params[:compute_profile_id].to_i,
                                 :compute_resource_id => params[:compute_resource_id].to_i)
   end
 
   def create
+    normalize_scsi_attributes(compute_profile_params["vm_attrs"]) if compute_attribute_params["vm_attrs"] && compute_attribute_params["vm_attrs"]["scsi_controllers"]
```

Comment:
this one is compute_profile_params but the next one is compute_attribute_params. which one is the right one?